build/run-backup.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+
+set -e
+
+request_backup() {
+	curl -s http://${SRC_NODE}:6033/backup/${BACKUP_NAME} -o /backup/${BACKUP_NAME}.stream
+}
+
+request_logs() {
+	curl -s http://${SRC_NODE}:6033/logs/${BACKUP_NAME}
+}
+
+run_s3() {
+	cat /backup/${BACKUP_NAME}.stream \
+		| xbcloud put ${BACKUP_NAME} --storage=s3 --s3-bucket="${S3_BUCKET}"
```

Comment:
I think we need to use almost the same options as for PXCO https://github.com/percona/percona-docker/blob/main/percona-xtradb-cluster-8.0-backup/run_backup.sh#L142